src/commands/Fun/pokemon.ts

```diff
@@ -0,0 +1,53 @@
+import { MessageType } from '@adiwajshing/baileys'
+import request from '../../lib/request'
+import MessageHandler from '../../Handlers/MessageHandler'
+import BaseCommand from '../../lib/BaseCommand'
+import WAClient from '../../lib/WAClient'
+import { IParsedArgs, ISimplifiedMessage } from '../../typings'
+import axios from 'axios'
+
+export default class Command extends BaseCommand {
+    constructor(client: WAClient, handler: MessageHandler) {
+        super(client, handler, {
+            command: 'pokemon',
+            aliases: ['pkmn'],
+            description: 'Gives you the data of the given pokemon. ',
+            category: 'weeb',
+            
+            usage: `${client.config.prefix}pokemon [name]`
+        })
+    }
+    // static count = 0
+    run = async (M: ISimplifiedMessage, { joined }: IParsedArgs): Promise<void> => {
+        
+        if (!joined) return void M.reply('Do you want me to give you the data of an unknown pokemon, Baka!')
+        const chitoge = joined.trim()
+        console.log(chitoge)
+        const { data } = await axios.get(`https://pokeapi.co/api/v2/pokemon/${chitoge}`)
```

Comment:
![image](https://user-images.githubusercontent.com/46681482/135766442-a444a3cd-7959-4fa5-b923-a014bad602d3.png)
This can return 

![image](https://user-images.githubusercontent.com/46681482/135766418-0924793a-8856-4d49-8f40-49763e90ee34.png)
If the pokemon query passed is an invalid pokemon.
Add a check for this too.